### ISSUE TITLE
Primal Blessing (ZG fist set)

### ISSUE THREAD

**Issue by Babygrim** [slowtorta/turtlewow-bug-tracker#4606]
The set bonus provides a proc, which when occurs overwrites orc racial

**Steps to reproduce the behavior with as much detail as possible:**

1. Equip Zul'Gurub set of fist weapons
2. Wait for it to proc
3. Use Orc racial

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

Item: https://classicdb.ch/?item=19896
Spell: https://classicdb.ch/?spell=20572

**Expected behavior. Describe how it should work:**
For example, I have 1000 Base Attack Power (means without any buffs), after I use orc racial i will have 1250, and if Set of Fists procs on top of that I should have 1550, but I only get to 1300. Instead of adding up, Fists proc overwrites orc racial AP bonus.

**Comment by Babygrim** [comment]
P.S. The Fist Set proc rate is actually low aswell

**Comment by balakethelock** [comment]
#3994

**Comment by Babygrim** [comment]
> #3994

So, orc racial sums up with Earthstrike trinket from Silithus badge quests, but not with ZG fists?

**Comment by balakethelock** [comment]
Of course, it's a bug that should be fixed high prio. Just linking other bug reports that are relevant to it to track everything together

**Comment by Babygrim** [comment]
> Of course, it's a bug that should be fixed high prio. Just linking other bug reports that are relevant to it to track everything together

Ohh, I missunderstood
